packages/logger/README.md

```diff
@@ -0,0 +1,48 @@
+<h1 align="center">@saasquatch/logger</h1>
+<p align="center">Thin wrapper around Winston to provide unified logging across SaaSquatch NodeJS services</p>
+<p align="center">
+  <a href="https://www.npmjs.com/package/@saasquatch/logger"><img src="https://img.shields.io/npm/v/@saasquatch/logger/latest.svg?style=flat-square" alt="NPM version" /> </a>
+  <a href="https://www.npmjs.com/package/@saasquatch/logger"><img src="https://img.shields.io/npm/dm/@saasquatch/logger.svg?style=flat-square" alt="NPM downloads"/> </a>
+</p>
+
+The SaaSquatch logging package facilitates unified structured logging across all
+NodeJS-based SaaSquatch services. It wraps Winston and provides standard log formats for
+general logging, debugging, and HTTP services.
+
+## Getting Started
+
+Install the package from NPM:
+
+```bash
+npm install @saasquatch/logger
+```
+
+Basic usage:
+
+```typescript
+import { initializeLogger, getLogger } from "@saasquatch/logger";
+
+function main() {
+  const logger = initializeLogger();
+
+  logger.info("Hello");
+  otherFunction();
+}
+
+function otherFunction() {
+  getLogger()!.info("World!");
+}
```

Comment:
Not sure I'm a huge fan of this API.

1. Why should I have to call `initializeLogger` if I'm not changing the default config? Maybe there's a default logger always set up?
2. `getLogger` wouldn't need the `!` if there was a default logger
3. `getLogger` is kind of awkward - if there's only one logger allowed at a time maybe you could provide helpers like `import { log, debug, info, warn}` etc so you can just do `log("debug", "hello")` or `debug("hello")`?

Reply:
> 1. Why should I have to call initializeLogger if I'm not changing the default config? Maybe there's a default logger always set up?
> 2. getLogger wouldn't need the ! if there was a default logger

I can add a default logger to resolve this if the single extra line to initialize the logger is too much boilerplate. Personally I think the original API is harder to misuse and it's more clear about what's actually happening behind the scenes.

> 3. getLogger is kind of awkward - if there's only one logger allowed at a time maybe you could provide helpers like import { log, debug, info, warn} etc so you can just do log("debug", "hello") or debug("hello")?

I can also implement the convenience imports, although personally I don't like these shorthand functions because they make the code less greppable (grepping for `logger.debug` is a lot more specific than `debug`).